core/mgmt/v1beta/mgmt.proto

```diff
@@ -119,12 +126,49 @@ message User {
   optional string role = 12 [(google.api.field_behavior) = OPTIONAL];
   // This defines whether the user is subscribed to Instill AI's newsletter.
   bool newsletter_subscription = 13 [(google.api.field_behavior) = REQUIRED];
-  // Console cookie token.
-  optional string cookie_token = 14 [(google.api.field_behavior) = OPTIONAL];
   // Profile Avatar in base64.
   optional string profile_avatar = 15 [(google.api.field_behavior) = OPTIONAL];
   // Profile Data.
   optional google.protobuf.Struct profile_data = 16 [(google.api.field_behavior) = OPTIONAL];
+  // Onboarding Status.
+  OnboardingStatus onboarding_status = 17 [(google.api.field_behavior) = OPTIONAL];
+}
+
+// User describes an individual that interacts with Instill AI.
+message User {
```

Comment:
I see many fields overlap between `User` and `AuthenticatedUser`. Would it make sense to have something like the following? 
```proto
message AuthenticatedUser {
  User public_user_info = 1;
  string email = 2;
  // ... other private fields
} 
```
Or even

```proto
message User { // User is the entity that contains the private fields
  PublicUserInfo public_user_info = 1;
  string email = 2;
  // ... other private fields
} 
```

I know it implies more changes as several endpoints already use `User`

Reply:
That's a good suggestion. Let me think.

Reply:
I'll update this in next PR along with the Stripe information refactor.